.gitignore

```diff
@@ -0,0 +1 @@
+.vs
```

Comment:
it is better to use `.vs/` instead. There is a difference between `.vs` and `.vs/`. Please google